lib/Doctrine/ruleset.xml

```diff
@@ -205,6 +205,8 @@
                 <element value="~^(User|Date|Time): \S+\z~i"/>
                 <element value="~^\S+ [gs]etter\.\z~i"/>
                 <element value="~^Class \S+\z~i"/>
```

Comment:
```suggestion
                 <element value="~^(Class|Interface|Trait) \S+\z~i"/>
```

Reply:
That is probably better, ive consolidated the regexes into one